flexget/plugins/modify/sort_by_weighted.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+from datetime import datetime, timedelta
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+__author__ = 'andy'
+
+log = logging.getLogger('sort_by_weighted')
+
+SUPPORTED_TYPES = (
+    int,
+    float,
+    bool,
+    datetime,
+    timedelta
+)
+
+ENTRY_NAME = 'sort_by_weight_sum'
+DEFAULT_SLOTS = 10
+
+
+class PluginSortByWeighted(object):
+    """
+    Sort task entries based on multiple fields using a sort weight per field.
+    Result per entry is stored in 'sort_by_weight_sum'.
+
+    Basic Concept:
+    For each field we calculate a weight based on given parameters and than sum the weights up and do a sort based on it.
+
+    field:          Name of the sort field
+    weight:         The sort weight used, values between 10-200 are good starts
+    weight_default: The default weight used if a sort 'field' could not be found or had a invalid entry (default is: 0)
+    inverse:        Use inverse weighting for the field, example: Date/Age fields
```

Comment:
Inverse weighting means that the lowest result slot gets the highest score and the highest gets the lowest score. So for Age/Date fields the entries with (0 days) get the maximum weight and older entries start getting lower and lower weights.